keps/sig-api-machinery/1040-priority-and-fairness/README.md

```diff
@@ -3013,10 +3013,10 @@ GA:
 
 - [x] API annotations properly support strategic merge patch.
 - [x] Design and implement borrowing between priority levels.
-- [ ] Satisfaction with LIST and WATCH support.
-- [ ] Implement borrowing by exempt priority levels.
-- [ ] APF allows us to disable client-side rate limiting without causing the apiservers to wedge/crash.  Note that there is another level of concern that APF does not attempt to address, which is mismatch between the throughput that various controllers can sustain.
-- [ ] Satisfaction that the interface is sufficient to support tuning of capacity and resource costs.
+- [x] Satisfaction with LIST and WATCH support.
```

Comment:
Yesterday we discussed that this is more of a "is it good enough" list -- is it worth documenting "Post GA" list of things we want to address but doesn't necessarily block GA? Specifically thinking about:
* LIST work estimator not accounting for namespaces
* Possibly better default values for the  default priority levels
* others?

Reply:
I chose to not try to list post-GA work, because I think we have not tried to agree on it and I do not think that is necessary to get to GA (which is the primary motivation of this PR).